ground/src/main/java/com/google/android/ground/model/submission/DateResponse.kt

```diff
@@ -31,8 +31,6 @@ data class DateResponse(val date: @Contextual Date) : Response {
     private val dateFormat: @Contextual DateFormat =
         SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
 
-    override fun getSummaryText(): String = detailsText
```

Comment:
Could you kindly file a cleanup bug to get rid of `getDetailsText()` as well? That should be handle in the view layer, where we're able to correctly localize the response depending on its type and the user's locale.

Reply:
Done, filed #1240